src/chart-review/chart-review.component.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { Route, Link, useHistory, useParams } from "react-router-dom";
+import SummariesNav from "../summaries/summaries-nav.component";
+import LevelTwoRoutes from "./level-two-routes.component";
+import styles from "./chart-review.css";
+
+export default function ChartReview(props: any) {
+  const [selected, setSelected] = React.useState();
+  let history = useHistory();
+  const [lastRoute, setLastRoute] = React.useState(history.location.pathname);
+  const [paths, setPaths] = React.useState({
+    summaries:
+      history.location.pathname.indexOf("summaries/") > -1
+        ? history.location.pathname
+        : ""
+  });
+
+  let { patientUuid } = useParams();
+
+  React.useEffect(() => {
+    const p = paths;
```

Comment:
why not just use paths directly? why reassign it to `p`?

Reply:
No good reason. Changed.